static/js/common/upload-addon.js

```diff
@@ -327,12 +327,28 @@
 
                     $("<strong>").text(message).appendTo(upload_results);
 
+                    // Specific messages for unlisted addons.
+                    var isListed = $('#id_is_listed').is(':checked'),
+                        isSideload = $('#id_is_sideload').is(':checked');
+                    if (!isListed) {
+                      if (isSideload) {
+                        $("<p>").text(gettext("Your submission, as a sideloaded add-on, will go through a manual review.")).appendTo(upload_results);
```

Comment:
I did use the term "side-loaded" in one of my notes in the submission flow mocks, for the checkbox that appears in the beginning of the process. I don't know if that's the best term to use, but I haven't heard other suggestions.

I don't know when this shows up, but I think the "as a sideloaded add-on" clause can be dropped.